### DISCUSSION TITLE
miwifix connected standby

### DISCUSSION THREAD

**Discussion by Quoose** [linux-surface/linux-surface#472] [Q&A]
I figure it's an ACPI thing. But wifi reconnects after standby defying the objective of connected standby. 
I'd like for a setting for it to remain active (partially because windows is stupid and refuses to change this ACPI setting after rebooting.)

**Comment by qzed** [comment]
IIRC there's some way to enable that via module options. Not sure if it works properly/has any bugs though. Cc @jonas2515.

**Reply by Quoose** [reply]
It seems it is set to remain connected by default on my Ubuntu stable. However it does not. I'm considering filing a bug, any suggestions?

**Reply by jonas2515** [reply]
There's the `disconnect_on_suspend` module parameter of mwifiex for this. If you set that to false, the card won't auto-disconnect when suspending.

Keep in mind there's still layers above the kernel like NetworkManager which will also try to disconnect before suspending, so you'd have to disable that, too.
